BenchmarkSample/ui/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is it possible to remove this file if it's a no-op?

Reply:
From other samples, looks like convention is to leave it in place, since studio generates it by default.